packages/doenetml-iframe/src/index.tsx

```diff
@@ -244,24 +241,27 @@ export function DoenetViewer({
     }
 
     return (
-        <iframe
-            ref={ref}
-            srcDoc={createHtmlForDoenetViewer(
-                id,
-                doenetML,
-                doenetViewerProps,
-                standaloneUrl,
-                cssUrl,
-            )}
-            style={{
-                width: "100%",
-                boxSizing: "border-box",
-                overflow: "hidden",
-                border: "none",
-                minHeight: 200,
-            }}
-            height={height}
-        />
+        <React.Fragment>
+            <OutsideIframeKeyboard />
```

Comment:
Maybe rename this `ExternalVirtualKeyboard`?

Reply:
done